### ISSUE TITLE
AttributeError in on_tag

### ISSUE THREAD

**Issue by jodal** [mopidy/mopidy#1449]
Running Mopidy 2.0.0  and playing music from Mopidy-Spotify 3.0.0.

``` python
Traceback (most recent call last):
  File "/home/jodal/mopidy-dev/mopidy/mopidy/audio/actor.py", line 225, in on_message
    self.on_tag(taglist)
  File "/home/jodal/mopidy-dev/mopidy/mopidy/audio/actor.py", line 340, in on_tag
    if self._audio._tags.get(key, unique) != value:
AttributeError: 'NoneType' object has no attribute 'get'
Traceback (most recent call last):
  File "/home/jodal/mopidy-dev/mopidy/mopidy/audio/actor.py", line 225, in on_message
    self.on_tag(taglist)
  File "/home/jodal/mopidy-dev/mopidy/mopidy/audio/actor.py", line 340, in on_tag
    if self._audio._tags.get(key, unique) != value:
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by jodal** [comment]
I've seen this again today, when changing from the first to the second Spotify track in the tracklist.

**Comment by SergioKob** [comment]
Hi Jodal,
I'm facing this too but it happens on every change, it doesn't stops or create any problem to mopidy/mopify, I think, do you have any solution?
BR

**Comment by jjok** [comment]
Hi. I'm seeing this same issue while using the Deezer extension.

**Comment by adamcik** [comment]
Fixed by e594d56
